toolkit/scripts/mariner-required-configs.json

```diff
@@ -824,6 +824,162 @@
                 "PR": [
                     "https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/a9ad05e35412cfa69ff93603500e97b7f29a90ca"
                 ]
+            },
+            "CONFIG_MELLANOX_PLATFORM": {
+                "value": [
+                    "y"
+                ],
+                "arch": [
+                    "ARM64"
+                ],
+                "comment": "Needed for Mellanox DPUs",
+                "PR": [
+                    "https://github.com/microsoft/CBL-Mariner/pull/5788"
```

Comment:
I think this should be https://github.com/microsoft/CBL-Mariner/pull/5789?

Reply:
Right, I add the review I created before, will fix.